scripts/buildLingeling

```diff
@@ -34,3 +34,8 @@ else
     (cd $ROOT/lingeling && ./configure.sh && make --silent && ./lingeling --version)
     echo "cmd: (cd $ROOT/lingeling && ./configure.sh && make --silent && ./lingeling --version) Done."
 fi
+
+# Copy $ROOT/lingeling/lingeling into CFI lib dir
+echo "Copying $ROOT/lingeling/lingeling into $ROOT/checker-framework-inference/lib"
```

Comment:
instead of downloading to `$ROOT/lingeling/lingeling` and then creating an extra copy in `CFI/lib`, please directly download to `CFI/lib` (change line 23)

in particular, pls download (line 27) the tar (wget command) to CFI/lib and then extract the tar to CFI/lib/lingeling